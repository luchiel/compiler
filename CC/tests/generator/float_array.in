void main()
{
double a[5];
a[0] = 0.0; a[1] = 4.4; a[2] = -0.6; a[3] = 44.8; a[4] = 3.2;

int j;
for(int i = 0; i < 4; ++i)
  a[i] += a[i + 1] + 2.3;

for(int i = 4; i >= 0; --i)
  printf("%.1f ", a[i]);
}