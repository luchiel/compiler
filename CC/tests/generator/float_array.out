3.2 50.3 46.5 6.1 6.7 